dev/tests/hu.elte.txtuml.api.model.execution.tests/src/hu/elte/txtuml/api/model/execution/unittests/NotBoundErrorTest.java

```diff
@@ -0,0 +1,22 @@
+package hu.elte.txtuml.api.model.execution.unittests;
+
+import hu.elte.txtuml.api.model.execution.seqdiag.sequences.SequenceBase;
+import hu.elte.txtuml.api.model.execution.testmodel.seqdiag.A;
+import hu.elte.txtuml.api.model.execution.testmodel.seqdiag.TestSig;
+import hu.elte.txtuml.api.model.impl.SequenceDiagramRelated;
+import hu.elte.txtuml.api.model.seqdiag.Proxy;
+import hu.elte.txtuml.api.model.seqdiag.Sequence;
+
+@SequenceDiagramRelated
+public class NotBoundErrorTest extends SequenceBase {
+	Proxy<A> r1;
```

Comment:
I think it is better to name the variable something like `proxy` to be meaningful.